### PR TITLE
Do not crash 'mgr-update-status' because 'long' type is not defined in Python 3

### PR DESCRIPTION
**Pull request by meaksh**
## What does this PR change?

This PR fixes an issue in `mgr-update-status` because of the use of type `long` which is not longer existing in Python 3:

```[Tue Mar 17 07:04:27 2020] up2date 
Traceback (most recent call last):
  File "/usr/sbin/mgr-update-status", line 55, in <module>
    cli.run()
  File "/usr/lib/python3.6/site-packages/up2date_client/rhncli.py", line 94, in run
    sys.exit(self.main() or 0)
  File "/usr/sbin/mgr-update-status", line 29, in main
    status_report = StatusCli.__build_status_report()
  File "/usr/sbin/mgr-update-status", line 48, in __build_status_report
    if status_report['uptime'][1] > long(2)**31-1:
<class 'NameError'>: name 'long' is not defined
```

This issue is currently making our cucumber testsuite to fail at `Scenario: Check spacewalk upd2date logs on client`

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: **bugfix**

- [x] **DONE**

## Test coverage
- No tests: **this is actually fixing a test failure**

- [x] **DONE**

## Links

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"		 
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by meaksh** [comment]
@juliogonzalez you may want to merge this in order to fix the only failure we have for `Uyuni:Master` in cucumber.

File: client/rhel/spacewalk-client-tools/src/bin/spacewalk-update-status.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def __build_status_report():
                 pass
 
         # We need to fit into xmlrpc's integer limits
-        if status_report['uptime'][1] > long(2)**31-1:
+        if status_report['uptime'][1] > int(2)**31-1:
```
Comment (mantel) [inline client/rhel/spacewalk-client-tools/src/bin/spacewalk-update-status.py]: Will this still work with python2? If yes, fine.

Comment (juliogonzalez) [reply client/rhel/spacewalk-client-tools/src/bin/spacewalk-update-status.py]: Tested (see comment below from @mantel itself) :-)

**Comment by mcalmer** [comment]
@mantel @meaksh - What about the second place in rhn_check.py ? AFAIK we copied the code.
If this fail, the other one should fail as well, right?

**Comment by mantel** [comment]
> @mantel @meaksh - What about the second place in rhn_check.py ? AFAIK we copied the code.
> If this fail, the other one should fail as well, right?

Argl, of course you are perfectly right! How could I already forget about this once again? @meaksh : There is 1:1 copy of the code which needs the same fix!

**Comment by meaksh** [comment]
Ha! This is not failing on `rhn_check` because of this: https://github.com/uyuni-project/uyuni/blob/master/client/rhel/spacewalk-client-tools/src/bin/rhn_check.py#L70

In any case, I think we should align those two codes. So I will create another PR for that :)

**Pull request merged**